components/form/AddNewItemForm.tsx

```diff
@@ -89,30 +94,44 @@ export default function AddNewItemForm({
           htmlFor='item_name'
           className='flex flex-col items-center gap-2 font-light'
         >
-          Item Name
+          <span className='flex items-center'>Item Name</span>
           <input
             type='text'
             className='input-text'
             {...register('item_name', { required: 'This field is required' })}
           />
         </label>
-        <p className='error-message'>{errors.item_name?.message}</p>
+        {(!itemNameValue || errors.item_name) && (
+          <p className='error-message'>
+            {errors.item_name?.message || 'This field is required'}
+          </p>
```

Comment:
We could maybe turn the pre-set message into an asterix by the field name and just show a message here if there is an error ?

Reply:
Could those two outliers not have their message hidden by default and just apply the styling we have for the rest in the event of an error?